packages/icons/icons.json

```
@@ -303,63 +303,63 @@
     {
         "displayName": "Inbox",
         "iconName": "inbox",
-        "tags": "folder, mail, file, message",
+        "tags": "folder, mail, file, message, drawer, container",
         "group": "file",
         "codepoint": 61951
     },
     {
         "displayName": "More",
         "iconName": "more",
-        "tags": "dots, three, extra, new, options",
+        "tags": "dots, three, extra, new, options, ellipses, break, pause, elipse, dot dot dot, suspense, see more, loading",
         "group": "interface",
         "codepoint": 62026
     },
     {
         "displayName": "Help",
         "iconName": "help",
-        "tags": "question mark, aid, advice, circle",
+        "tags": "question mark, aid, advice, circle, question",
         "group": "action",
         "codepoint": 61932
     },
     {
         "displayName": "Calendar",
         "iconName": "calendar",
-        "tags": "calendar, timeframe, agenda, diary, day, week, month",
+        "tags": "timeframe, agenda, diary, day, week, month",
         "group": "interface",
         "codepoint": 61758
     },
     {
         "displayName": "Media",
         "iconName": "media",
-        "tags": "audio, video, media, picture, image, drawing, illustration",
+        "tags": "audio, video, picture, image, drawing, illustration, mountain, photo",
         "group": "media",
         "codepoint": 62013
     },
     {
         "displayName": "Link",
         "iconName": "link",
-        "tags": "link, connection, network",
+        "tags": "connection, network, paperclip, attach",
```

```suggestion
        "tags": "connection, network, chain, attach",
```

it's not a paperclip, it's a link in a chain